templates/bundle_version.sh

```diff
@@ -0,0 +1,16 @@
+GIT=$(sh /etc/profile; which git)
```

Comment:
Ah, OK. This has more UPPERCASE variable names than I usually see, but that's not really a huge deal.